### PR TITLE
activate transifex and improve travis

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,13 +37,9 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git $HOME/maintainer-quality-tools
```
Comment (tafaRU) [inline .travis.yml]: Could you please add `--depth=1`?
See https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L26 as reference.

**Comment by pedrobaeza** [comment]
Please add also runbot badge on README.

**Comment by eLBati** [comment]
@tafaRU @pedrobaeza done, thanks

**Comment by tafaRU** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1: 

Merging then

**Pull request merged**